test/helpers/deploy.js

```diff
@@ -9,9 +9,12 @@ const Neumark = artifacts.require("Neumark");
 const Curve = artifacts.require("Curve");
 const TestCommitment = artifacts.require("TestCommitment");
 const WhitelistedCommitment = artifacts.require("WhitelistedCommitment");
-const FeeDistributionPool = artifacts.require("FeeDistributionPool");
+const RoleBasedAccessControl = artifacts.require("RoleBasedAccessControl");
+const AccessRoles = artifacts.require("AccessRoles");
 
 export async function deployAllContracts(
+  lockAdminAccount,
```

Comment:
make it part of the config object (last argument)

Reply:
why? I need to send those with each call, they are not part of configuration but runtime

Reply:
Ohh shit. Github hid some files from me and i thought that this is somehow optional. Can you leave then 'invariant' call at the beginning of the function to throw if required args are not provided? example: https://github.com/Neufund/ico-contracts/blob/master/test/helpers/deploy.js#L61

Reply:
sure